src/main/java/io/fabric8/jenkins/openshiftsync/Lifecyclable.java

```diff
@@ -1,7 +1,9 @@
 package io.fabric8.jenkins.openshiftsync;
 
+import hudson.model.Descriptor.FormException;
+
 public interface Lifecyclable {
     public void stop();
-    public void start();
+    public void start() throws FormException;
```

Comment:
I don't love the idea of adding `throws FormException` to this interface. Looking at the JavaDoc, this appears to be an error due to a Jenkins form throwing some kind of (validation?) error: https://javadoc.jenkins.io/hudson/model/Descriptor.FormException.html

Can we find the root cause where this Exception could be thrown, and add a `try / catch` blocks around those specific locations?